Provide more guidance (#302)
etsap-TIMES/xl2times: README.md

make html
```

## Testing on an existing model

> If you have an exsiting TIMES model in Excel (e.g. developed using Veda) and would like to use the tool with it, we recommend to conduct bulk testing first. Bulk testing will allow understanding how much of the syntaxis used in the model is supported by the tool.

Reviewer: ```suggestion
If you have an existing TIMES model in Excel (e.g. developed using Veda) and would like to use the tool with it, we recommend to conduct bulk testing first. Bulk testing will allow understanding how much of the syntax used in the model is supported by the tool.
```

> Start by generating `*.dd` files based on `AllScenario` scenario group (i.e. in Veda).

Reviewer: ```suggestion
Start by generating `*.dd` files based on `AllScenario` scenario group (i.e. in Veda), and place them in a directory called `ground_truth`.
```

Author: "placing in ground_truth" is done by the command below, is it not? I.e. no need for the user to do anything?

Reviewer: Oh sorry, my bad! Ignore this

> Afterwards, execute the following command from the root of the tool (assumes `My_Bulk_Test` case name) to exctract all the data from the `*.dd` files:

Reviewer: ```suggestion
Afterwards, execute the following command from the root of the tool (assumes `My_Bulk_Test` case name) to extract all the data from the `*.dd` files:
```

Reviewer: Does it need to be from the root directory of this repository? Wouldn't `python /path/to/xl2times/dd_to_csv.py` work? Also, isn't it `utils/dd_to_csv.py`?

Author: It could be `python /path/to/xl2times/xl2times/dd_to_csv.py` as well... I think it would fit better under `utils`, but, no, it is under `xl2times`...

```bash
python xl2times/dd_to_csv.py "C:\VEDA\GAMS_WrkTIMES\My_Bulk_Test" ground_truth
```

Finally, execute the tool on the model (e.g. `My_TIMES-Model`) and compare the results to the previously extracted data:
```bash

> xl2times "C:\VEDA\VEDA_Models\My_TIMES-Model"  --ground_truth_dir=ground_truth

Author: @siddharth-krishna do you think we should mention that this assumes that the virtual environment has been activated before?

Reviewer: Oh yes, that's a good idea
